docs/README.md

```diff
@@ -0,0 +1,1423 @@
+<!-- Generated with Stardoc: http://skydoc.bazel.build -->
+
+<!-- Edit docs/README.md.tpl and run `bazel run //docs:update-readme` to change the project README. -->
+
+# Nixpkgs rules for Bazel
+
+[![Build status](https://badge.buildkite.com/79bd0a8aa1e47a92e0254ca3afe5f439776e6d389cfbde9d8c.svg?branch=master)](https://buildkite.com/tweag-1/rules-nixpkgs)
+
+Use [Nix][nix] and the [Nixpkgs][nixpkgs] package set to import
+external dependencies (like system packages) into [Bazel][bazel]
+hermetically. If the version of any dependency changes, Bazel will
+correctly rebuild targets, and only those targets that use the
+external dependencies that changed.
+
+Links:
+* [Nix + Bazel = fully reproducible, incremental
+  builds][blog-bazel-nix] (blog post)
+* [Nix + Bazel][youtube-bazel-nix] (lightning talk)
+
+[nix]: https://nixos.org/nix
+[nixpkgs]: https://github.com/NixOS/nixpkgs
+[bazel]: https://bazel.build
+[blog-bazel-nix]: https://www.tweag.io/posts/2018-03-15-bazel-nix.html
+[youtube-bazel-nix]: https://www.youtube.com/watch?v=hDdDUrty1Gw
+
+## Rules
+
+* [nixpkgs_git_repository](#nixpkgs_git_repository)
+* [nixpkgs_local_repository](#nixpkgs_local_repository)
+* [nixpkgs_package](#nixpkgs_package)
+* [nixpkgs_cc_configure](#nixpkgs_cc_configure)
+* [nixpkgs_cc_configure_deprecated](#nixpkgs_cc_configure_deprecated)
+* [nixpkgs_java_configure](#nixpkgs_java_configure)
+* [nixpkgs_python_configure](#nixpkgs_python_configure)
+* [nixpkgs_go_configure](#nixpkgs_go_configure)
+* [nixpkgs_rust_configure](#nixpkgs_rust_configure)
```

Comment:
Looks like Go and Rust are missing. At least their links are not active.

Reply:
This is very weird. I will have to pull apart the docs-specific repository rules first, to see what the difference to the other toolchains even is. They build individually, and I wouldn't know why in this collection anything would change.

Reply:
Ah, we did not have them in `//nixpkgs:nixpkgs.bzl` to avoid strict dependencies on Go and Rust tooling, and so there are no aliases. Alright, this goes hand with the other changes.

Reply:
The Go one seems to be still missing, it's not contained in the toplevel `README`. [The diff](https://github.com/tweag/rules_nixpkgs/pull/202/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L1411) shows it as removed.

I noticed that the symbol is missing in the list in `docs/BUILD.bazel`, but the following patch didn't change things:
```
diff --git a/docs/BUILD.bazel b/docs/BUILD.bazel
index 24e46ed..6a14150 100644
--- a/docs/BUILD.bazel
+++ b/docs/BUILD.bazel
@@ -11,6 +11,7 @@ generate_documentation(
         "nixpkgs_cc_configure_deprecated",
         "nixpkgs_java_configure",
         "nixpkgs_python_configure",
+        "nixpkgs_go_configure",
         "nixpkgs_rust_configure",
         "nixpkgs_sh_posix_configure",
     ],
```

Reply:
Yes, this is intended, see https://github.com/tweag/rules_nixpkgs/commit/d55b95f9672b4ebaf6540abf6c69a873e2ece4f6. If we generated Go in the top-level `README` we either would have to alias `nixpkgs_go_configure` in `nixpkgs/nixpkgs.bzl`, which would force Go dependencies on unsuspecting users; or complicate the build machinery disproportionately. I decided to just link to the submodule.

Reply:
I see, makes sense to go this route.